examples/sub-module/bundles-size-KBs.csv

```diff
@@ -0,0 +1,28 @@
+name - bundled - minified
+IPFS - 1685.41 - 661.64
+add - 968.11 - 344.13
+bitswap - 684.80 - 232.13
+block - 724.77 - 250.23
+bootstrap - 685.64 - 232.39
+cat - 725.05 - 250.33
+commands - 683.89 - 231.76
+config - 686.87 - 233.10
+dht - 688.51 - 233.41
+diag - 684.97 - 232.18
+files - 1120.38 - 404.30
+get - 907.74 - 318.35
+id - 684.31 - 231.95
+key - 684.59 - 232.03
+log - 685.40 - 232.38
+ls - 684.00 - 231.80
+mount - 684.18 - 231.86
+name - 684.63 - 232.06
+object - 923.66 - 340.57
+pin - 685.51 - 232.39
+ping - 684.59 - 231.90
+pubsub - 740.40 - 249.54
+refs - 684.39 - 231.94
+repo - 684.56 - 232.02
+swarm - 1324.18 - 527.03
+update - 684.45 - 231.96
+version - 684.21 - 231.88
```

Comment:
Please use commas in the comma separated values files :) 

Also, add the units to the top.